src/main/resources/db/changelog/20230220150433-drop-act-key-env-tables.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20230220150433-1" author="wpoteat">
+        <comment>
+            Remove old activation key and environment tables
+        </comment>
+
+        <sql>
+            DROP TABLE cp2_activation_key_products;
+            DROP TABLE cp2_environment_content;
```

Comment:
It might be a good idea to add preconditions to make sure the current state of the db is what we expect.